File: ProApi.App/src/app/_models/Speaker.ts
import { SocialNetwork } from './SocialNetwork';
import { Event } from './Event';

export interface Speaker {
  id: number;
  name: string;
  miniCurriculo: string;
  imageUrl: string;
  phone: string;
  email: string;
  socialNetworks: SocialNetwork[];
  speakerEvents: Event[];
}

File: ProApi.Domain/Speaker.cs
using System.Collections.Generic;

namespace ProApi.Domain
{
    public class Speaker
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string MiniCurriculo { get; set; }
        public string ImageUrl { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public List<SocialNetwork> SocialNetworks { get; set; }
        public List<SpeakerEvent> SpeakerEvents { get; set; }
    }
}